docs/content/preview/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/on-premises-manual.md

```diff
@@ -697,6 +560,20 @@ As an alternative to setting crontab permissions, you can install systemd-specif
     WantedBy=default.target
     ```
 
+### Ulimits on Red Hat Enterprise Linux 8
+
+On Red Hat Enterprise Linux 8-based systems (Red Hat Enterprise Linux 8, Oracle Enterprise Linux 8.x, Amazon Linux 2), additionally, enter the following commands:
+
+```sh
+vi /etc/systemd/system.conf 
```

Comment:
Same here, we can have a ```sh``` part to show the line DefaultLimitNOFILE=1048576 but then just list the two files they line need to be in